core/base-service/base.js

```diff
@@ -114,7 +114,7 @@ class BaseService {
    * credentials to the request. For example:
    * - `{ options: { auth: this.authHelper.basicAuth } }`
    * - `{ options: { headers: this.authHelper.bearerAuthHeader } }`
-   * - `{ options: { qs: { token: this.authHelper.pass } } }`
```

Comment:
Unrelated to Twitch, but a similar piece of code (or rather documentation) that was missed in #4729.